layouts/partials/header.html

```diff
@@ -42,12 +42,16 @@
                             <a class="navbar-link" href="#">{{ .Name }}</a>
                             <div class="navbar-dropdown">
                                 {{ range .Children }}
+                                {{ if strings.HasPrefix .URL "https" }}
+                                <a class="navbar-item" href="{{ .URL | relURL }}" target="_blank" rel="noopener">{{ .Name }}</a>
```

Comment:
This adds `target` and `rel` attributes for one link on the menu. What we need is something that will add this for all external links regardless of where they are rendered from.

[The issue](https://issues.apache.org/jira/browse/CAMEL-13888) in question has some pointers on how to accomplish that.

Reply:
Great. Will fix that today. Thanks for the pointers